Dockerfile

```diff
@@ -1,9 +1,19 @@
 FROM python:3.6.3-stretch
 
 RUN apt-get -y update \
- && apt-get -y install wget unzip build-essential cmake pkg-config libatlas-base-dev gfortran \
-                       libgtk2.0-dev libavcodec-dev libavformat-dev libswscale-dev libjpeg-dev libpng-dev \
-                       libtiff-dev libv4l-dev \
+  && apt-get -y install autoconf-archive automake g++ libtool pkg-config wget unzip build-essential cmake \
```

Comment:
Can we do some cleanup here? Since curl will work for downloading we could remove wget. Also should remove these build dependencies once we are done with them